checkstyle.xml

```diff
@@ -161,7 +161,6 @@
         <module name="EqualsHashCode"/>
         <module name="IllegalInstantiation"/>
         <module name="InnerAssignment"/>
-        <module name="MagicNumber"/>
```

Comment:
Put this back. You reduced the number of magic numbers so you only have one now, but in FantasyLeagueService, you should have a private static final int WEEKS_IN_MONTH = 4 and use that on line 179

Reply:
okay!

Reply:
I would also put a private static final int MIN_GAMES_IN_FANTASY_WEEK= 4 for the first 4 in the if statement